src/pages/Home/home.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Header, HeroImage, Footer, WhyUsSection, Categories, Deals, Trending } from '../../components'
+
+import './home.css'
+
+function HomePage(){
+    return(
+        <div className="home-bg">
+            <Header />

```

Comment:
if header and footer is not changing on routing then you can use it from outside the route

Reply:
Sure.